python/cuml/cuml/dask/ensemble/base.py

```diff
@@ -402,6 +394,8 @@ def _func_predict_proba_partial(model, input_data, **kwargs):
     than dataset.
     """
     X = concatenate(input_data)
+    if "convert_dtype" in kwargs:
+        del kwargs["convert_dtype"]
```

Comment:
Can we drop that earlier?

Reply:
Turns out this line is not needed at all, now that FIL supports `convert_dtype`